Src/IronPython/Modules/Builtin.cs

```diff
@@ -1503,6 +1503,11 @@ [NotNull]IList enumerator
             return value;
         }
 
+        /// <summary>
+        /// Calls the __next__ method of an iterable
+        /// </summary>
+        /// <param name="iter">Iterable instance</param>
+        /// <returns>Next object or throws an StopIterator exception</returns>
         public static object next(IEnumerator iter) {
             if (iter.MoveNext()) {
```

Comment:
No, cause this is the `Builtin next` method, which is can be called directly as a function.